content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,145 @@
+{{< figure link="/img/blogs/newsletter-103/after-balloon.png" src="/img/blogs/newsletter-103/after-balloon.png" caption="AFTER" >}}
+{{< /gallery >}}
+
+You can track most of his Shell related work [here](https://github.com/reactos/reactos/pulls?q=is%3Apr+author%3Acbialorucki+is%3Aclosed).
+
+## Initial DirectX work-ground
+Oleg Dubinskiy is working on squashing bugs that would not allow games to properly use the DirectX kernel subsystem (`dxg.sys` and `dxgkrnl.sys` respectively).
+More specifically he fixed some issues in the Win32 subsystem kernel backend (`win32k.sys`) where DirectX was not loaded hence the impossibility of accessing its APIs for use ([#4551](https://github.com/reactos/reactos/pull/4551)).
+
+This fixes an issue where it was not possible to change the display mode in VMware as it attempts to invoke the DirectDraw API to handle the display management.
+Along with it, he worked on implementing DirectDraw management code in display mode switch ([#4519](https://github.com/reactos/reactos/pull/4519)).
+This improves the experience with games that are based upon DirectX stack.
+
+In the meantime Justin Miller is syncing up all of the DirectX DLLs with Wine up to the latest version ([#5688](https://github.com/reactos/reactos/pull/5688)).
+
+## NT6+ application compatibility preparations
+A lot of applications that we take for granted, be it games, browsers, E-mail clients, utilities and such rely on newer APIs provided by Windows.
+NT6+ is a terminology that encompasses Windows operating systems such as Vista, 7, 8, 8.1, 10 and 11 that are based on a newer NT architecture design.
+At the API compatibility level, ReactOS currently aims for Windows Server 2003, which had an NT release version of 5.2; the last iteration in the NT5 family of Windows operating systems.
+
+As many software developers no longer support older versions of Windows, most of the applications that we use day to day will not work on ReactOS.
+However, this is going to change.
+A group made up of Timo Kreuzer, Justin Miller, and other developers and contributors alike are developing the necessary APIs for compatibility with modern programs.
+While Timo is still working on implementing a dynamic versioning system for DLLs ([#3239](https://github.com/reactos/reactos/pull/3239)) that allows exporting of routines to applications depending on their compatibility settings, he has added the option to ReactOS bot builders to compile builds with NT6 exports which makes it possible to experiment with NT6+ application compatibility.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-103/obs-nt6.png" src="/img/blogs/newsletter-103/obs-nt6.png" caption="OBS 20.1 running on ReactOS (with some glitches)" >}}
+{{< figure link="/img/blogs/newsletter-103/obs-stream.png" src="/img/blogs/newsletter-103/obs-stream.png" caption="Streaming ReactOS running OBS 20.1" >}}
+{{< /gallery >}}
```

Comment:
Weird, but the last image shows shortened compared to what it could be:
![image](https://github.com/reactos/web-content/assets/1969829/17af9a7c-26c1-4f1a-850a-21652dc006e5)

I wonder whether it's posible to tweak this gallery/figure thing in order to make it show something like that instead:
![image](https://github.com/reactos/web-content/assets/1969829/40cdb77a-1fc4-4e5d-a209-8fd89c8b813b)

Reply:
This has to be tweaked in the CSS styles somehow.